client/shared/src/main/scala/org/http4s/client/oauth1/package.scala

```diff
@@ -76,7 +72,7 @@ package object oauth1 {
       nonceGenerator: F[Nonce],
       callback: Option[Callback] = None,
       verifier: Option[Verifier] = None
-  )(implicit F: MonadThrow[F], W: EntityDecoder[F, UrlForm]): F[Request[F]] =
+  )(implicit F: Async[F], W: EntityDecoder[F, UrlForm]): F[Request[F]] =
```

Comment:
Thinking about what it means when a constraint is strengthened solely because one platform requires it.  I guess keeping the public APIs aligned is probably worth it.

Reply:
Yeah, this is a tricky one :(

Reply:
I'm a bit squeamish about "strongest constraint any platform needs."  If a future platform required stronger constraints following this rule, it's a breaking change to everybody.  But I can't imagine adding a third platform anytime soon, so this is an academic concern.

On the other hand, it's nice to present as uniform an interface as possible for things advertised as the same artifact.  Cross-builders on top of us would be in for a bad surprise if they build on `MonadThrow`, add scala.JS later, and find they need an `Async`.  Everything downstream would have to diverge in the same ways.

Yeah, I think you chose the lesser evil.